perceval/backends/jira.py

```diff
@@ -163,6 +167,34 @@ def parse_issues(raw_page):
         for issue in issues:
             yield issue
 
+    @staticmethod
```

Comment:
It's ok for me, but also all those kind of functionalities in other backends are still in the `Backend` class (maybe not as functions, but logic is there) , so I'm afraid we are breaking the structure followed until now.